Samples/tutorials/excel-tutorial/README.md

```diff
@@ -0,0 +1,34 @@
+# Excel Tutorial - Completed
+
+This sample is the result of completing the [Tutorial: Create an Excel task pane add-in](https://learn.microsoft.com/office/dev/add-ins/tutorials/excel-tutorial). It was constructed with the [Yeoman generator for Office Add-ins](https://learn.microsoft.com/office/dev/add-ins/develop/yeoman-generator-overview).
+
+The tutorial gives step-by-step instructions on how to add functionality alongside explanations as to why code is being added. This sample is best used if you've completed the tutorial and want to start from a more complete project. It's also helpful when debugging problems you encounter during the tutorial.
```

Comment:
```suggestion
The tutorial gives step-by-step instructions on how to add functionality alongside explanations as to why code is being added. Use this sample if you want to explore and try the completed code, or if you need to debug any issues you may have encountered while following the tutorial.
```

Reply:
Just avoiding suggestion that you should do the tutorial first.

Reply:
But don't we want people to take the tutorial first? I think if someone had this PNP as their entry point, we'd want them to go back and follow the step-by-step instructions, unless they really don't want to.

Reply:
I'm assuming there's different learning styles and some developers may want to just run the code and step through or study it before doing all the copy/paste work from the tutorial.